packages/nextra/src/client/components/tabs/tabs.tsx

```diff
@@ -32,16 +33,18 @@ export const Tabs: FC<
     items: (TabItem | TabObjectItem)[]
     children: ReactNode
     storageKey?: string
-  } & Pick<TabGroupProps, 'defaultIndex' | 'selectedIndex' | 'onChange'> &
-    Pick<TabListProps, 'className'>
+    className?: TabListProps['className']
+    tabClassName?: HeadlessTabProps['className']
```

Comment:
```suggestion
    className: TabListProps['className']
    tabClassName: HeadlessTabProps['className']
```
they should already have `undefined` as possible value, also can you add a changeset? `patch`

Reply:
Won't this create issues in projects with `exactOptionalPropertyTypes: true`?

i've just made a CodeSandbox to show this: https://codesandbox.io/p/sandbox/f9y9cl

<img width="937" alt="image" src="https://github.com/user-attachments/assets/fb18ef92-2343-4826-bb49-1c34dc5e0092">